languages/js/src/Polar.ts

```diff
@@ -151,10 +151,13 @@ export class Polar {
       const query = this.#ffiPolar.nextInlineQuery();
       this.processMessages();
       if (query === undefined) break;
+      const source = query.source();
       const { results } = new Query(query, this.#host);
       const { done } = await results.next();
       results.return();
-      if (done) throw new InlineQueryFailedError(name);
+      if (done) {
+        throw new InlineQueryFailedError(source, name);
+      }
```

Comment:
This is the style we use across the rest of the lib
```suggestion
      if (done) throw new InlineQueryFailedError(source, name);
```